### ISSUE TITLE
Dynamic NFT Minting

### ISSUE THREAD

**Issue by earth2travis** [ourzora/ocs-hackathon-24#4]
Using the [🐸 frog](https://frog.fm/intents/button-mint) framework to create frames on Farcaster and we want to allow the user to mint a dynamic image based on their input.

Is it possible to mint the image generated by the frame to a collection on Zora?

**Comment by iainnash** [comment]
Yes it should be able to! You can upload to IPFS and insert IPFS URLs or use standard HTTP centralised URLs following the standard ERC1155 metadata format.

https://eips.ethereum.org/EIPS/eip-1155
